core/wallet/transaction.go

```diff
@@ -117,7 +129,7 @@ func DepositRequest(prepareResponse *exPb.PrepareDepositResponse, privateKey *ec
 }
 
 // Continuous call ConfirmDeposit until it responses a FAILED or SUCCESS.
-func ConfirmDepositProcess(prepareResponse *exPb.PrepareDepositResponse, privateKey *ecdsa.PrivateKey) {
+func ConfirmDepositProcess(n *core.IpfsNode, prepareResponse *exPb.PrepareDepositResponse, privateKey *ecdsa.PrivateKey) {
 	ctx := context.Background()
```

Comment:
can you double check all wallet operations involve exchange calls should have ctx passed down (req.Context()) from the command, only use context.Background() if it's called from goroutine. I think we missed this in past reviews, check all deposit/withdrawal/confirm commands.

Reply:
okay, will do.